### PR TITLE
Updated Estuary Performance Dashboard Page V1

### PR DESCRIPTION
**Pull request by APiligrim**
This pr updates Estuary's performance page with new logos and updated styling. I kept the darker theme for "dark mode" but later on there would be light mode as well similar to estuary's home page.

Follow ups I want to make to improve it:
- [ ] resize smaller and bigger logos individually to best fit the grid 
- [ ] create a smoother scrolling behavior for anchors
- [ ] create a collapsable menu for mobile
- [ ] improve the "On Chain" and "Sealed" design

Screenshots: 
Estuary_ Ecosystem dashboard  (4)

Mobile:

<p float="left">
  
   
</p>

### CODE REVIEW

File: pages/index.module.scss [hunk 1]
```diff
@@ -352,17 +369,191 @@ a.flink {
...
+  background: var(--main-page-background-dark-mode);
+  padding-left: 10vw;
+  padding-right: 10vw;
 }
 
-.ecosystemParagraph {
-  color: rgba(128, 128, 128, 1);
-  padding: 0 24px 0 24px;
-  font-size: 24px;
+.ecosystemEstuaryLogo{
```
Comment (jimmylee) [inline pages/index.module.scss:377]: NIT: Rewrite as such

```
.ecosystemEstuaryLogo{
  padding-left: 8.4vw;
  position: absolute;
  top: 50%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  
  @media (max-width: 600px) {
    padding-left: 5vw;
    top: 40px;
    padding-left: 6vw;
  }
}
```

This is the same thing

File: pages/index.module.scss [hunk 2]
```diff
@@ -352,17 +369,191 @@ a.flink {
...
+}
+
+@media (max-width: 598px){
+  .ecosystemNavLink{
+    font-size: 20px;
+  }
+}
+
+@media (max-width: 952px){
+  .ecosystemNav .ecosystemNavList .ecosystemNavListItem .ecosystemNavLink{
```
Comment (jimmylee) [inline pages/index.module.scss:497]: See previous nit and fix

File: pages/index.module.scss [hunk 3]
```diff
@@ -371,13 +562,14 @@ a.flink {
   display: inline-flex;
 }
 
-.ecosystemStatCard {
-  padding: 20px;
-}
-
 .ecosystemStatValue {
+  color: white;
```
Comment (jimmylee) [inline pages/index.module.scss:566]: Why not variable :-)

**Review by jimmylee — APPROVED** [review]
LGTM, some things for you to fix though in a follow up.

**Comment by alvin-reyes** [comment]
Did we intentionally remove the Environment cost? I guess we should put that somewhere else?

**Comment by alvin-reyes** [comment]
Did we intentionally remove the Environment cost? I guess we should put that somewhere else?

**Pull request merged**